src/unstract/sdk/index.py

```diff
@@ -144,34 +144,32 @@ def extract_text(
         """
         self.tool.stream_log("Extracting text from input file")
         extracted_text = ""
+        x2text = X2Text(
```

Comment:
The init call to X2Text can also raise X2TextError... Should we be handling that also? 
https://github.com/Zipstack/unstract-sdk/blob/171470c5bed8b30ed0c7c7f62841b4ef38f075b3/src/unstract/sdk/x2txt.py#L46

Reply:
This will be raised as an `X2TextError` and will be appended to error messages framed by the callers up the call stack